tests/messages/koji/build_building.json

```diff
@@ -0,0 +1,78 @@
+{
+    "body":{
+        "headers":{
+            "content-length": "840", 
+            "expires": "0", 
+            "old": "null", 
+            "JMS_AMQP_MESSAGE_FORMAT": "0", 
+            "timestamp": "0", 
+            "JMS_AMQP_NATIVE": "false", 
+            "destination": "/topic/VirtualTopic.eng.brew.build.building", 
+            "priority": "4", 
+            "subscription": "/queue/Consumer.client-datanommer.upshift-prod.VirtualTopic.eng.>", 
+            "message-id": "ID:messaging.domain.com-37550-1534679656045-5:49075:0:0:1", 
+            "version": "4.3", 
+            "attribute": "state", 
+            "release": "17.el8", 
+            "new": "BUILDING", 
+            "JMS_AMQP_FirstAcquirer": "false", 
+            "type": "BuildStateChange", 
+            "name": "squashfs-tools"
+        },
+        "msg":{
+            "info": {
+              "package_name": "squashfs-tools", 
```

Comment:
Could you make this consistent with the other messages you added to be `e2e` (and fix the other values in this message)?